api/envoy/config/endpoint/v3/load_report.proto

```diff
@@ -48,6 +50,29 @@ message UpstreamLocalityStats {
   // upstream endpoints in the locality.
   uint64 total_issued_requests = 8;
 
+  // The total number of connections in an established state at the time of the
+  // report. This information may be based on upstream_cx_active metric
```

Comment:
Maybe point out that these references to metrics are Envoy specific - they wouldn't apply on gRPC or other dataplanes. I'd be personally fine leaving them in the PR comment history or Envoy-specific docs somewhere as well. If we keep them here, ideally use proper RST formatting for metric references, e.g. ```
``upstream_cx_active_metric``
```